### DISCUSSION TITLE
atoti covid example

### DISCUSSION THREAD

**Discussion by athos-liu** [atoti/atoti#657] [Q&A] [answered]
I'm watching [the atoti tutorial on youtube](https://www.youtube.com/watch?v=uIeSpiK7JF8&ab_channel=atoti) , the source data used, ie owid_covid_data.csv , may I ask where can I get a copy? In [owid website COVID-19 Data Explorer](https://ourworldindata.org/explorers/coronavirus-data-explorer) , on the right-bottom there's a "Download" button, where I can get a csv file, is that the same one?

**Comment by bloussou** [comment, marked as answer]
Hello,
I think that is the right one !
